packages/apps/plugins/plugin-markdown/src/MarkdownPlugin.tsx

```diff
@@ -196,7 +196,7 @@ export const MarkdownPlugin = (): PluginDefinition<MarkdownPluginProvides> => {
               if (isDocument(data.active)) {
                 const { readonly } = settings.values.state[data.active.id] ?? {};
                 return (
-                  <MainLayout toolbar={settings.values.toolbar}>
```

Comment:
Please restore this — `MainLayout` needs the `toolbar` flag to apply the correct layout when the toolbar is present.